kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -158,4 +158,12 @@ object HighAvailabilityConf {
       .version("1.5.0")
       .stringConf
       .createOptional
+
+  val DISCOVERY_CLIENT_CLASS: ConfigEntry[String] =
+    buildConf("kyuubi.service.discovery.client.class")
+      .doc("Class name for service discovery client.")
+      .version("1.6.0")
+      .stringConf
+      .checkValue(_.nonEmpty, "must not be empty")
```

Comment:
use classOf[DiscoveryClient].isAssignableFrom(cls)

Reply:
Here it is a string config, better avoid loading the class and check isAssignableFrom